order-book/src/main/java/com/market/banica/order/book/model/ItemMarket.java

```diff
@@ -0,0 +1,50 @@
+package com.market.banica.order.book.model;
+
+import com.market.Origin;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Component
+public class ItemMarket {
+
+    private final Map<String, TreeSet<Item>> allItems;
+
+    @Autowired
+    public ItemMarket() {
+        allItems = new ConcurrentHashMap<>();
+        addDummyData();
+    }
+
+    public Set<Item> getItemSetByName(String itemName) {
+        return allItems.get(itemName) == null ? Collections.emptySet() : allItems.get(itemName);
```

Comment:
Please consider getOrDefault instead of all of the checks

Reply:
Updated to getOrDefault().